get_data/graphs_for_server.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pandas as pd
+import matplotlib
+from matplotlib import pyplot as plt
```

Comment:
Flake8 would complain at your to order these alphabetically, but since thats's currently broken would you mind changing this now to reduce the amount of linting errors that will need fixing in one go please :)